src/main/resources/swagger/api-docs.yaml

```diff
@@ -808,3 +851,12 @@ definitions:
       enabled:
         $ref: '#/definitions/Enabled'
         description: the status of the user's account
+
+  WorkbenchEmail:
```

Comment:
WorkbenchEmail should look just like a string when rendered as json, not an object with a value property

Reply:
actually, this applies to all the "value" objects. They don't get rendered in json as an object with a value property. They look just like strings in json.